docs/README.md

```diff
@@ -30,6 +30,18 @@ For configuration examples for the caddy plugin look at our [examples section](/
 ### Location
 The TXT record must be accessible under the subdomain "\_redirect".
 
+### URLs
+* Please encode all URLs in the TXT record since the parser expects encoded URLs. In some cases, unencoded characters may break your configuration.
+* Most notably, ";" should be escaped as "%3B" in a URL since the parser will split the URL on the unencoded semicolon.
+
+Examples: <br />
+; -> %3B <br />
+/ -> %2F <br />
+? -> %3F <br />
+= -> %3D <br />
+
+*https://example.com/over/there?name=ferret -> https%3A%2F%2Fexample.com%2Fover%2Fthere%3Fname%3Dferret*
```

Comment:
Maybe the users don't need to escape *everything* including the `https://` part and the `/` characters; we still want the records to be readable

Reply:
Sounds good - I'm removing the URL scheme and `/` from what should be encoded. Should we add something to the docs that says that a `/` doesn't need to be encoded?